eng/readme-templates/README.md

```diff
@@ -88,6 +88,6 @@ See the [.NET Framework Lifecycle FAQ](https://support.microsoft.com/help/17455/
 
 # License
 
-* Legal Notice: [Container License Information](https://aka.ms/mcr/osslegalnotice)
-* The .NET Framework images use the same license as the [Windows Server Core base image](https://hub.docker.com/_/microsoft-windows-servercore/).
-* The [Visual Studio Tools license](https://visualstudio.microsoft.com/license-terms/mlt031519/) also applies to the [SDK images](https://hub.docker.com/_/microsoft-dotnet-framework-sdk/).
+* The [MICROSOFT SOFTWARE SUPPLEMENTAL LICENSE FOR WINDOWS CONTAINER BASE IMAGE](https://hub.docker.com/_/microsoft-windows-servercore/) applies to [].NET Framework container images](https://hub.docker.com/_/microsoft-dotnet-framework/).
```

Comment:
```suggestion
* The [MICROSOFT SOFTWARE SUPPLEMENTAL LICENSE FOR WINDOWS CONTAINER BASE IMAGE](https://hub.docker.com/_/microsoft-windows-servercore/) applies to [.NET Framework container images](https://hub.docker.com/_/microsoft-dotnet-framework/).
```

Reply:
Thoughts on saying `applies to all` to help provide clarification with the second bullet.

Reply:
I'm sorry Matt, my last comment was clear as mud!  I intended the `applied to all` to be added to the first bullet.